src/components/ReportActionItem/ReportPreview.js

```diff
@@ -97,11 +98,13 @@ const defaultProps = {
 function ReportPreview(props) {
     const reportAmount = CurrencyUtils.convertToDisplayString(ReportUtils.getMoneyRequestTotal(props.iouReport), props.iouReport.currency);
     const managerEmail = props.iouReport.managerEmail || '';
-    const managerName = ReportUtils.isPolicyExpenseChat(props.chatReport) ? ReportUtils.getPolicyName(props.chatReport) : ReportUtils.getDisplayNameForParticipant(managerEmail, true);
+    const managerAccountID = props.iouReport.managerID || 0;
+    const managerName =
+        (ReportUtils.isPolicyExpenseChat(props.chatReport) ? ReportUtils.getPolicyName(props.chatReport) : ReportUtils.getDisplayNameForParticipant(managerAccountID, true)) || managerEmail;
```

Comment:
do you need () around ReportUtils.isPolicyExpenseChat(props.chatReport)?

Reply:
Seems like it

Reply:
Seems like it